docs/openapi/components/schemas/credentials/CertificateOfOrigin.yml

```diff
@@ -2,6 +2,8 @@ $linkedData:
   term: CertificateOfOrigin
   '@id': https://w3id.org/traceability#CertificateOfOrigin
 title: Certificate Of Origin
+tags:
+  - eCommerce
```

Comment:
This is used in OG, steel, Ecommerce, and AG how do we want to handle tagging for a schema that is used in all?
I'm proposing we add all tags ( obviously out of scope for this PR)

Reply:
Multiple tags can be added with

```
tags:
  - eCommerce
  - Steel
  - OG
  - AG
```